src/framework/core/js/FluidDocument.js

```diff
@@ -174,4 +174,17 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         };
     });
 
+    /* Sets the value to the DOM element and triggers the change event on the element.
+     * Note: when using jQuery val() function to change the node value, the change event would
+     * not be fired automatically, it requires to be initiated by the user.
+     *
+     * @param node {A jQueryable DOM element} A selector, a DOM node, or a jQuery instance
+     * @param value {String|Number|Array} A string of text, a number, or an array of strings
+     * corresponding to the value of each matched element to set in the node
+     */
+    fluid.changeElementValue = function (node, value) {
```

Comment:
Hi @cindyli - yes, sorry for my forgetfulness that I had already recommended the creation of this utility - this is fine